src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (x) => {
+    return (y) => {
+      count++;
+
+      if (count <= 3 || (count % 2)) {
+        return x + y;
+      } else {
+        return 'Bzzz... Error!';
+      }
```

Comment:
In my honest opinion this is a redundant else.
```suggestion
      }
      
      return 'Bzzz... Error!';
```

Reply:
agree, else is redundant here